stories/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "extends": "../tsconfig.json",
     "include": ["../src/typings.d.ts", "./*.ts", "./*.tsx"],
     "exclude": ["src/**/*.spec.ts"],
+    "files": [""],
```

Comment:
Why is this needed?

Reply:
Well, i can't really explain why this works. But it stopped random build fails. When error message was something like `Files is empty blah blah` i think it's storybook quirk